docs/sp-add-ins/sharepoint-admin-apis-authentication-and-authorization.md

```diff
@@ -0,0 +1,62 @@
+---
+title: SharePoint admin APIs authentication and authorization
+description: This article describes the authentication and authorization model for SharePoint admin APIs.
+author: samkabue
+ms.author: samkabue
+ms.date: 06/28/2024
+ms.localizationpriority: high
+ms.service: sharepoint
+---
+
+# SharePoint admin APIs authentication and authorization
+
+SharePoint admin APIs are accessible via [SharePoint client object model (CSOM)](./complete-basic-operations-using-sharepoint-client-library-code.md), [SharePoint REST service](./get-to-know-the-sharepoint-rest-service.md), and [Microsoft Graph](../apis/sharepoint-rest-graph.md).
+
+## Overview
+
+Here are some key principles about SharePoint admin APIs authentication and authorization:
+
+- Admin APIs can be called by administrators (including users with [granular delegated admin privileges (GDAP)](/partner-center/gdap-supported-workloads#sharepoint)) and applications registered in Microsoft Entra.
+- Each admin API defines the least-permissive permission a caller should have to call the API.
+- We use role-based access control to assert a caller can call a particular admin API. If the caller has not been granted the required permission, they'll get an unauthorized access error.
+
+## Pre-Requisites
+
+- Caller is a user with an assigned administrator role in Microsoft Entra, or
```

Comment:
Reading this again, it sounds like the caller is either a user or an application. In reality, it is an app+user or an app-only. But it never is just a user.

Reply:
As discussed offline, removing the user-only scenario since that auth path is getting deprecated soon.